_docs/developer/xdebug.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Xdebug Setup Instructions
+category: Developer
+---
+
+Xdebug is a PHP extension automatically enabled in the Vagrant VM that allows for step-by-step code debugging, code profiling, and code tracing. For more information about the features of Xdebug, see [2.xdebug.org](https://2.xdebug.org/).
```

Comment:
I think you want the url https://xdebug.org/, the one you linked has a danger banner saying it'll be discontinued